lua/telescope/builtin/files.lua

```diff
@@ -209,6 +209,17 @@ files.find_files = function(opts)
     ),
     previewer = conf.file_previewer(opts),
     sorter = conf.file_sorter(opts),
+    attach_mappings = function(prompt_bufnr, map)
+
+      local change_working_directory = function ()
+        local entry = action_state.get_selected_entry(prompt_bufnr).value
+        local dir = vim.fn.fnamemodify(entry, ":p:h")
+        vim.fn.execute("lcd " .. dir, "silent")
```

Comment:
I changed this to lcd as suggested, but I just realized lcd only changes it inside the telescopeprompt buffer and not the active buffer. Will change it back to cd